library/src/main/java/com/chuckerteam/chucker/api/config/Feature.kt

```diff
@@ -0,0 +1,14 @@
+package com.chuckerteam.chucker.api.config
+
+import android.content.Context
+import androidx.annotation.StringRes
+import androidx.fragment.app.Fragment
+
+interface Feature {
+    @get:StringRes
+    val name: Int
+    val tag: Int
```

Comment:
Tag is not really clear here what exactly it is

Reply:
I've renamed it to `id`, is it clearer? Each feature should have a unique `id`.